src/cmd_line.rs

```diff
@@ -120,6 +145,15 @@ pub struct CmdLineSettings {
         default_value = "neovide"
     )]
     pub x11_wm_class_instance: String,
+
+    /// Set background to light, dark, or match system setting with "auto"
+    #[arg(
+        long = "theme",
+        env = "NEOVIDE_THEME",
+        value_parser = EnumValueParser::<ThemeChoice>::new(),
+        required = false
+    )]
+    pub theme: Option<ThemeChoice>,
```

Comment:
`required` is not required (ironic) if the target type is `Option`. clap knows automagically that any `Option`al arguments are not required (if unset).

```suggestion
    )]
    pub theme: Option<ThemeChoice>,
```